changelog/27889.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+secrets/database: Add PSC support for GCP CloudSQL MySQL and Postgresql also add PrivateIP support for MySQL
+```
```

Comment:
I suggest making 2 `release-note:improvement` entries in this file

Reply:
I am having trouble formatting an example in GH markdown. Here is an example gist: https://gist.github.com/fairclothjm/b77cabf9682fd6e591da57f977aa22aa

Reply:
Updated